app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx

```diff
@@ -41,5 +40,62 @@ export const ipAddressStyle = css`
 `;
 
 export const connectionPanelMainContainer = css`
-	margin-top: ${token('space.200', '16px')};
+	margin-top: ${token('space.200')};
+
+	[role=tab] {
+		&:first-of-type {
+			padding-left: ${token('space.0')};
+
+			::after {
+				margin-left: -50px !important;
```

Comment:
Doesn't look like token/space works with negative spacing 🤔 And the only way I could override the styling of the tab components to get the alignment correct was to use !important.

Reply:
/nit Have you tried `space.negative.[token here]`? https://atlassian.design/foundations/spacing#negative-values

Could you add an id and target that instead of using important?